wp-content/themes/humanity-theme/styles/blue.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemas.wp.org/wp/6.4/theme.json",
```

Comment:
I've used the 6.4 version of the JSON schema, as this makes it easier to use theme.json settings that are supported up to a specific version of WordPress.